eng/restore/binplacePackages/Configurations.props

```diff
@@ -13,6 +13,7 @@
       netstandard;
       netcoreapp2.0;
       netcoreapp3.0;
+      netcoreapp;
```

Comment:
@ericstj I thought sourcebuild doesn't allow this?

Reply:
`netstandard` will also be [restored](https://github.com/dotnet/corefx/blob/6549a93fb462f581696af11a3be076d36309dc7e/eng/restore/dirs.proj#L5) for source build.  Source build will need to add [stand-ins](https://github.com/dotnet/source-build-reference-packages) for these packages, as is the case for any of the packages which we dead-end (we won't be the only repo that needs to consume them).

Reply:
Don't you want to revert this change?

Reply:
yes this should be removed.